spec/use_cases/reservations/cancel_spec.rb

```diff
@@ -1,19 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe Reservations::UseCases::Cancel do
-  describe ".call" do
+  describe '.call' do
+    let(:seats) { [*'10'..'13'] }
     let(:movie) { create(:movie) }
     let(:hall) { create(:hall) }
+    let(:user) { create(:user) }
     let(:screening) { create(:screening, hall:, movie:, start_time: Date.current + 2.days) }
-    let(:reservation) { create(:reservation, screening:, status: :booked) }
-    let(:params) do 
-      { id: reservation.id }
+    let(:reservation) do
+      Reservations::UseCases::Create.new(screening_id: screening.id,
+                                         seats:,
+                                         user_id: user.id,
+                                         status: :booked).call
```

Comment:
we never create variables like that :D You don't now if using this use case doesn't create any errors. You definitely should use factory bot like before. Looking at your new factory it could be simply 
```suggestion
let(:reservation) { create(:reservation) }
```

Reply:
i changed a way of validating reservation model. I'm validating it after creating tickets for it so factory wasn't working anymore because it raised an active record error connected to seats not being chosen and reservation itself does not hold information about seats. I used that use case as a not so elegant workaround but now I changed factory for reservation and I hope it makes sense and is acceptable.